spec/tests/manual.fmf

```diff
@@ -1,50 +1,16 @@
-summary: Manual test execution details
+summary: Test automation state
 
 story:
-    As a tester I want to store detailed manual instructions
-    covering test scenario so that I can perform them manually
-    before the test can be fully automated.
+    As a tester I need to create test which is not automated.
```

Comment:
Could we probably re-use/adjust the story description here? The short version does not describe anything about the motivation.

Reply:
Sure, I'm trying to remove promise of automation as it might not happen (at it doesn't matter).